Aikido.Zen.Core/Helpers/RouteHelper.cs

```diff
@@ -362,7 +368,12 @@ private static bool IsAllowedExtension(string segment)
             if (!string.IsNullOrEmpty(extension))
             {
                 extension = extension.TrimStart('.');
-                if (extension.Length >= 2 && extension.Length <= 5 || IgnoreExtensions.Contains(extension))
+                if (extension.StartsWith("asp", StringComparison.OrdinalIgnoreCase))
```

Comment:
ends with?

Reply:
it's "extension"

Reply:
Ah yes, thought it was the route itself, cool!